utxo/utxo.go

```diff
@@ -623,9 +623,12 @@ func (uv *UtxoVM) SelectUtxos(fromAddr string, fromPubKey string, totalNeed *big
 			}
 			txInputs = append(txInputs, txInput)
 			cacheKeys[uKey] = true
-			if utxoTotal.Cmp(totalNeed) >= 0 {
-				foundEnough = true
-				break
+			// when cancel lease, totalNeed equals negative one
+			if totalNeed != big.NewInt(-1) {
```

Comment:
is there a better way?

Reply:
Thank you for your advice and I will correct it.